### ISSUE TITLE
Nested scrolling in sticker picker

### ISSUE THREAD

**Issue by Bubu** [turt2live/matrix-dimension#362]
Sometimes there are stacked scrollbars in the dimension stickerpicker:

![image](https://user-images.githubusercontent.com/105185/86521716-b0258280-be54-11ea-9ac9-94c53ee159af.png)

This happens when the sticker-pack tooltip is long enough to reach into the out-of-scroll-view area to the right (see screenshot).

**Comment by TheTimeWalker** [comment]
Yup, I noticed this happening when implementing this bottom bar but I never came to fixing that, sorry!
